common/test/acceptance/pages/lms/conditional.py

```diff
@@ -0,0 +1,42 @@
+"""
+Conditional Pages
+"""
+from bok_choy.page_object import PageObject
+
+POLL_ANSWER = 'Yes, of course'
+
+
+class ConditionalPage(PageObject):
+    """
+    View of conditional page.
+    """
+
+    url = None
+
+    def is_browser_on_page(self):
+        """
+        Returns True if the browser is currently on the right page.
+        """
+        return self.q(css='.xblock-student_view').present
```

Comment:
The xblock-student_view is not specific to the conditional module, is it? Is there any class that is specific to the conditional module that can be used?